src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java

```diff
@@ -0,0 +1,114 @@
+    private static final Logger LOG = LoggerFactory.getLogger(NotificationUtils.class);
+    private static final String SPARKPOST_KEY = getConfigPropertyAsText("SPARKPOST_KEY");
+    private static final String FROM_EMAIL = getConfigPropertyAsText("NOTIFICATION_FROM_EMAIL");
+
+    /**
+     * Send a SMS message to the provided phone number
+     * @param toPhone - e.g., +15551234
+     * @param body - SMS message body
+     * @return messageId if message was sucessful (null otherwise)
+     */
+    public static String sendSMS(String toPhone, String body) {
+        if (ACCOUNT_SID == null || AUTH_TOKEN == null) {
+            LOG.warn("SMS notifications not configured correctly.");
+            return null;
+        }
+        try {
+            Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
+            PhoneNumber fromPhoneNumber = new PhoneNumber(FROM_PHONE);
+            PhoneNumber toPhoneNumber = new PhoneNumber(toPhone);
+            Message message = Message.creator(
+                toPhoneNumber,
+                fromPhoneNumber,
+                body
+            ).create();
+            LOG.debug("SMS ({}) sent successfully", message.getSid());
+            return message.getSid();
+        } catch (Exception e) {
+            LOG.error("Could not create SMS", e);
+            return null;
+            // FIXME bugsnag
+        }
+    }
+
+    /**
+     * Send notification email using Sparkpost.
+     * TODO: determin if we should use sparkpost or sendgrid.
+     */
+    public static void sendEmail(String toEmail, String subject, String text, String html) {
+        if (SPARKPOST_KEY == null || FROM_EMAIL == null) {
+            LOG.warn("Notifications disabled due to invalid config. Skipping message to {} SUBJECT: {}", toEmail, subject);
+            return;
+        }
+        Client client = new Client(SPARKPOST_KEY);
+        try {
+            Response response = client.sendMessage(FROM_EMAIL, toEmail, subject, text, html);
+            LOG.info(response.getResponseMessage());
+        } catch (Exception e) {
+            // FIXME: Add bugsnag
+            LOG.error("Could not send notification to {}", toEmail, e);
+        }
+    }
+
+    /**
+     * Send notification email using Sendgrid.
+     * TODO: determin if we should use sparkpost or sendgrid.
+     */
+    public static boolean sendSendGridEmail(String toEmail, String subject, String text, String html) {
+        String SENDGRID_API_KEY = getConfigPropertyAsText("SENDGRID_API_KEY");
+        if (SENDGRID_API_KEY == null || FROM_EMAIL == null) {
+            LOG.warn("Notifications disabled due to invalid config. Skipping message to {} SUBJECT: {}", toEmail, subject);
+            return false;
+        }
+        Email from = new Email(FROM_EMAIL);
+        Email to = new Email(toEmail);
+        Content content = new Content("text/plain", text);
+        Mail mail = new Mail(from, subject, to, content);
+
+        SendGrid sg = new SendGrid(SENDGRID_API_KEY);
```

Comment:
Seems like this could be initialized upon application load in a static block of code.